lib/components/user/saved-trip-screen.js

```diff
@@ -91,29 +96,46 @@ class SavedTripScreen extends Component {
   }
 
   _updateMonitoredTrip = async () => {
-    const { accessToken, persistence, wizard } = this.props
+    const { accessToken, isCreating, persistence } = this.props
 
     if (persistence && persistence.otp_middleware) {
       const { monitoredTrip } = this.state
 
       // TODO: Change state of Save button.
 
       let result
-      if (wizard) {
+      if (isCreating) {
         result = await addTrip(persistence.otp_middleware, accessToken, monitoredTrip)
       } else {
         result = await updateTrip(persistence.otp_middleware, accessToken, monitoredTrip)
```

Comment:
A lot of the code in this function should be handled in the redux store. Please move it there. This code looks very similar to the changes for user management before it was fixed. We need to be consistent with the patterns we're following.

Reply:
How about this approach?
- Create a field `state.user.loggedInUser.monitoredTrips` to store all monitored trips for the user.
- Load the monitored trips upon user log in.
- Implement the URL scheme #/savedtrips/{tripId}

Reply:
`state.user.loggedInUser.monitoredTrips` won't work, but `state.user.loggedInUserMonitoredTrips` will probably do.